lms/static/sass/shared/_course_object.scss

```diff
@@ -180,78 +191,116 @@
         }
       // }
 
-      .info {
-        background: $content-wrapper-bg;
-        height: 220px + 130px;
-        left: 0px;
-        position: absolute;
-        top: 0px;
-        @include transition(all 0.15s linear 0s);
-        width: 100%;
-        overflow: hidden;
-
+      .course-image {
         .cover-image {
-          height: 200px;
+          height: 160px;
           overflow: hidden;
-          width: 100%;
+
+          &:before {
+            @include fade-transition();
+            content: '';
+            position: absolute;
+            top: 0;
+            left: 0;
+            background: black;
+            opacity: 0;
+            width: 100%;
+            height: 160px;
+            z-index: 10;
+          }
+
+          &:after {
+            @include fade-transition();
+            @include box-sizing(border-box);
+            @include calc( top, '80px - 25px');
+            @include calc( left, '50% - 100px');
+            content: 'LEARN MORE';
+            position: absolute;
+            width: 200px;
+            height: 50px;
+            z-index: 20;
+            border: 3px solid white;
+            border-radius: 3px;
+            font-family: $sans-serif;
+            font-weight: $font-semibold;
+            font-size: 1.1em;
+            line-height: 44px;
+            color: white;
+            padding: 0 20px;
+            text-align: center;
+            opacity: 0;
+          }
 
           img {
-            display: block;
-            min-height: 100%;
             width: 100%;
           }
         }
+      }
 
-        .desc {
-          @include box-sizing(border-box);
-          height: 120px;
-          overflow: hidden;
-          padding: 10px 10px 12px 10px;
-          position: relative;
-          width: 100%;
+      .course-info {
+        height: 200px;
 
-          section {
-            color: $base-font-color;
-            font: normal 1em/1.6em $serif;
-            margin: 0;
-            height: 100%;
-            overflow: hidden;
-            text-overflow: ellipsis;
-          }
+        .course-organization, .course-code, .course-date {
+          color: #3d3e3f;
+          font-size: 0.9em;
```

Comment:
Please use a typography Sass extend if possible when defining font-size/weight - https://github.com/edx/edx-platform/blob/mjevtic/SOL-531/lms%2Fstatic%2Fsass%2Felements%2F_typography.scss.

If you need a specific font-size/setting that the list does not support, please use our ``font-size()`` mixin (e.g. (``@include font-size(18)``)).